### PR TITLE
mvebu: armada-385-linksys-rango device tree fixes

### PR DESCRIPTION
**Pull request by nitroshift**
Disable sdhci node as it is not used in this device, fix regulator gpio address and also the power, e-sata and wireless leds.

### CODE REVIEW

File: target/linux/mvebu/files/arch/arm/boot/dts/armada-385-linksys-rango.dts [hunk 1]
```diff
@@ -340,17 +330,28 @@
...
+                        gpios = <&gpio1 13 GPIO_ACTIVE_LOW>;
+                };
+ 
+                wlan_5g {
+                        label = "rango:white:wlan_5g";
+                        gpios = <&gpio1 14 GPIO_ACTIVE_LOW>;
+                };
+ 
+                power {
+ 			label = "rango:white:power";power {
```
Comment (thagabe) [inline target/linux/mvebu/files/arch/arm/boot/dts/armada-385-linksys-rango.dts]: i think you added a ":power";power {"

File: target/linux/mvebu/files/arch/arm/boot/dts/armada-385-linksys-rango.dts [hunk 2]
```diff
@@ -424,23 +415,38 @@
...
-		marvell,pins = "mpp24", "mpp29";
+	sata_led_pin: sata-led-pin {
+		marvell,pins = "mpp21";
 		marvell,function = "gpio";
 	};
 
-	power_led_pin: power-led-pin {
-		marvell,pins = "mpp56";
+	wps_key_pin: wps-key-pin {
+		marvell,pins = "mpp24";
```
Comment (thagabe) [inline target/linux/mvebu/files/arch/arm/boot/dts/armada-385-linksys-rango.dts]: on LEDE the pins are "mpp24" tho im not sure which is correct

**Comment by thagabe** [comment]
@nitroshift 

9d28951 has already been applied with 6f4047d

**Comment by wigyori** [comment]
NAK. This doesn't work like this that you close the PR and resubmit. You review the PR and suggest/request changes, and let the author do it.

**Pull request closed without merging**